single_sync_v3.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# if $2 is not empty, set a variable DRY to true
+if [ $2 = true  ]; then
+  DRY=true
+  echo "  - DRY MODE: $DRY"
+else
+  DRY=false
+  echo "  - DRY MODE: $DRY"
+fi
+
+IMAGES=$(yq e '.images | length' $1)
+for (( c=0; c<${IMAGES}; c++ ))
+do
+    NAME=$(yq e '.images['"${c}"'].name' $1)
```

Comment:
is this change in the index access (`["'"${c}"'"]`) regarding how it was done in v2 intended?:
```diff
16c16,20
<     MULTI_ARCH=$(yq e '.images['"${c}"'].multi-arch' $1)
---
>     MULTI_ARCH=$(yq e '.images["'"${c}"'"].multi-arch' "$1")
```